### PR TITLE
Add network-mux dependencies from Hackage

### PR DESCRIPTION
**Pull request by mark-stopka**
Add `network-mux` dependencies from Hackage:
  - abstract-deque
  - abstract-par
  - dense-linear-algebra
  - monad-par
  - monad-par-extras
  - optparse-generic
  - reducers
  - statistics
  - statistics-linreg
  - tdigest

Upstream [network-mux](https://github.com/input-output-hk/ouroboros-network/tree/master/network-mux), [OBS](https://build.opensuse.org/package/show/devel:languages:haskell:cardano/ghc-network-mux).

### CODE REVIEW

**Comment by mimi1vx** [comment]
@mark-stopka you accidentally removed `aeson` from ghc-9.0.x set

+ please squash commits to max 2 - one for ghc8 and second for ghc-9

File: src/cabal2obs/Config/Ghc810x.hs [hunk 1]
```diff
@@ -67,7 +67,9 @@ resolveConstraints = unwords ["cabal", "install", "--dry-run", "--minimize-confl
  -}
 
 constraintList :: ConstraintSet
-constraintList = [ "adjunctions"
+constraintList = [ "abstract-deque"
+                 ,"abstract-par"
```
Comment (peti) [inline src/cabal2obs/Config/Ghc810x.hs:71]: Please follow the formatting of the other entries.

File: src/cabal2obs/Config/Ghc90x.hs [hunk 2]
```diff
@@ -67,7 +67,9 @@ resolveConstraints = unwords ["cabal", "install", "--dry-run", "--minimize-confl
  -}
 
 constraintList :: ConstraintSet
-constraintList = [ "adjunctions"
+constraintList = [ "abstract-deque"
+                 ,"abstract-par"
+                 ,"adjunctions"
```
Comment (peti) [inline src/cabal2obs/Config/Ghc90x.hs:72]: ditto

**Review by peti — CHANGES_REQUESTED** [review]

File: src/cabal2obs/Config/Ghc810x.hs (hunk 1 above)
Comment (mark-stopka) [reply src/cabal2obs/Config/Ghc810x.hs:71]: Are yiu reffering to a missing `space` in front of `"abstract-par"`?

Comment (peti) [reply src/cabal2obs/Config/Ghc810x.hs:71]: Yes.

Comment (mark-stopka) [reply src/cabal2obs/Config/Ghc810x.hs:71]: Thx, that must've got there by total mistake... :-) sorry for inconvenience!

**Comment by mark-stopka** [comment]
@mimi1vx can you or @peti check this version? If ok, I would then squash and merge into a larger dependency set or multiple smaller sets depending on your preferences...

**Comment by mimi1vx** [comment]
LGTM ,squash?

**Comment by mark-stopka** [comment]
@mimi1vx done

**Pull request merged**